Update changelog (#255)
glotzerlab/signac: changelog.txt

Added
+++++

>  - ``Project.id`` and ``Job.id`` properties (#250)

Reviewer: ```suggestion
 - ``Project.id`` and ``Job.id`` properties (#250).
```

Deprecated
++++++++++

>  - get_id in ``Project`` and ``Job`` class (#250)

Reviewer: ```suggestion
 - get_id in ``Project`` and ``Job`` classes has been replaced in favor of the id property (#250).
```

Reviewer 2: Since the entries have a period at the end, they should probably be complete sentences as well. I am aware that this is not true for all previous entries.

>  - Deprecate the get_id method in ``Project`` and ``Job`` classes in favor of the id property, to be removed in 2.0 (#250)

Reviewer 2: ```suggestion
 - The ``get_id`` method in ``Project`` and ``Job`` classes in favor of the ``id`` property, to be removed in 2.0 (#250)
```

Author: @csadorf you commented that you would prefer full sentences. The suggested change makes the sentence a fragment. I don't mind, but is that what you prefer?

Reviewer 2: Sorry, I responded to an outdated version of the code. All I wanted to suggest were the added code markups.

>  - Deprecate the ``get_id`` method in ``Project`` and ``Job`` classes in favor of the ``id`` property, to be removed in 2.0 (#250)

Reviewer: ```suggestion
 - Deprecate the ``get_id`` method in ``Project`` and ``Job`` classes in favor of the ``id`` property, to be removed in 2.0 (#250).
```
